segments/weather.sh

```diff
@@ -160,6 +160,10 @@ __get_condition_symbol() {
 			#echo "﹌"
 			echo "〰"
 			;;
+        "breezy")
```

Comment:
The indentation is off, compare with the rest of the file :).  Make sure to use hard tabs for indentation.